package.json

```diff
@@ -49,33 +68,35 @@
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^14.0.4",
     "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.23",
-    "@types/react": "^17.0.43",
-    "@types/react-dom": "^17.0.14",
+    "@types/node": "^18.16.3",
+    "@types/react": "^18.2.5",
+    "@types/react-dom": "^18.2.3",
     "@types/reactstrap": "^8.7.2",
-    "@typescript-eslint/eslint-plugin": "^5.18.0",
-    "@typescript-eslint/parser": "^5.18.0",
-    "babel-loader": "^8.2.4",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
     "bootstrap": "^4.0.0",
-    "concurrently": "^7.6.0",
+    "concurrently": "^8.0.1",
     "cross-env": "^7.0.3",
-    "eslint": "^8.12.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-storybook": "^0.5.8",
-    "gh-pages": "^3.2.3",
+    "eslint": "^8.39.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-import-resolver-typescript": "^3.5.5",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-storybook": "^0.6.12",
+    "gh-pages": "^5.0.0",
     "jest": "^27.5.1",
-    "microbundle": "^0.14.2",
-    "nodemon": "^2.0.20",
-    "prettier": "^2.6.2",
+    "nodemon": "^2.0.22",
+    "prettier": "^2.8.8",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "reactstrap": "^8.9.0",
-    "release-it": "^15.7.0",
+    "reactstrap": "^9.1.9",
```

Comment:
@neotob 

Reactrap 9+ required bootstrap 5+. I would recommend updating to the latest minor version of version 8x

https://reactstrap.github.io/?path=/story/home-upgrading--page

Reply:
good one thank you, yeah first i went crazy on updating everything to the latest... and then realized package for package that i have to go some versions back to make it compatible.. must have miss this one thanks!

Reply:
Yeah i know the struggle 😄 

Usually I start with `npx npm-check-updates --target minor` to update all packages to the latest minor version, which shouldn't introduce any breaking changes. At least in theory ;-) 

And than I check which things can easily be upgraded to the latest major version. Like tools we only use for development etc.

Finally i check if the remaining packages make sense to update and possibly check if they have a migration guide.